### DISCUSSION TITLE
Error during initial migration when using a MySQL database

### DISCUSSION THREAD

**Discussion by roenfeldt** [thedevdojo/wave#151] [Q&A] [answered]
Hey Wave community :)

As the title says, I changed the database type from the default SQLite, into MySQL. I made sure the database exists, it is empty, and that the credentials are accurate in the .env file.

When running the initial `php artisan migrate` command, all the tables are successfully created up until the `2024_03_29_230138_create_subscriptions_table ` migration is reached, which fails throwing the following exception message:

```
SQLSTATE[HY000]: General error: 1824 Failed to open the referenced table 'plans' (Connection: mysql, SQL: alter table `subscriptions` add constraint `subscriptions_plan_id_foreign` foreign key (`plan_id`) references `plans` (`id`) on delete cascade)
```

I don't have any problems using the default SQLite database, but I'd still like to know how to get a successful initial migration in case I ever need to switch to a MySQL database.

For a deeper insight, I created the MySQL database with **TablePlus**, and my dev machine is an **M1 MacBook Pro** running **macOS Sequoia 15.1**.

I'd highly appreciate any help on this.

Many thanks!

**Comment by bobbyiliev** [comment, marked as answer]
Hey! 👋 

I believe that just answered the same question here: https://devdojo.com/question/wave-installation-failed

The new error that you are getting is fixed in:

> https://github.com/thedevdojo/wave/pull/138

You can cherry pick that commit or pull the latest changes from the Wave repo.

Let me know how it goes!

**Comment by roenfeldt** [comment]
@bobbyiliev pulling the latest changes from the Wave repo fixed it, thank you so much!

**Reply by bobbyiliev** [reply]
No worries! Happy to hear that!
